README.md

```diff
@@ -1,5 +1,35 @@
-# UKHO Docs
+# Software Engineering at the UKHO
 
-Documentation around digital development project, teams, behaviors and processes at the UKHO.
+This repo contains documentation about digital delivery teams, behaviors and processes at the UKHO.
 
-Users can find the documentation at https://docs.ukho.dev
+We have eight software engineering teams that develop and support software using the .Net and Java stacks.  
+
+We’re proud that we’ve created this group by recruiting people who share a commitment to professional development and watched them grow with us.   Right now, we have people pursuing certification in Azure, AWS, Kubernetes and Security; we have others on management training courses; we even have people on Masters Degree programmes. 
+
+
+## How we work
+
+Everything we do is team based.  We take Agile seriously; your delivery team will be small but will have the people it needs to get things done - developers, testers, product owner and a delivery manager - as well as access to infrastructure specialists, UX experts and analysts. 
+
+Quality is paramount.  Our [engineering](./docs/software-engineering-policies/README.md) and [quality assurance](./docs/quality-assurance/README.md) policies support our teams to do good work.  Most code is done [in pairs](./docs/software-engineering-policies/PairProgramming/PairProgrammingPolicy.md) or larger groups to ensure quality and spread knowledge.  Everyone [reviews code](./docs/software-engineering-policies/CodeReview/CodeReviewPolicy.md) and everyone welcomes feedback. 
+
+We strive to automate as much as possible: [testing](./docs/quality-assurance/test-strategy.md), builds, deployments use the latest automation tools available.
+
+We actively try and pay down [technical debt](./docs/software-engineering-policies/TechnicalDebt/TechnicalDebtGuidance.md).
+
+## Our community
```

Comment:
We use the terms "community", "Practice" and "Department" in the first two paragraphs - can we agree a common term to use.

Reply:
OK, fixed